### DISCUSSION TITLE
How can I navigate through multiple code sections on the Pybricks code editor in the LEGO Robot Inventor Hub using only the home button?

### DISCUSSION THREAD

**Discussion by AyadTatari** [pybricks/support#1393] [Q&A]
I have multiple codes and I want to navigate through them only by clicking the home button on the robot inventor hub how can I do that?

**Comment by laurensvalk** [comment]
> I have multiple codes and I want to navigate through them only by clicking the home button on the robot inventor hub how can I do that?

If you'd like to make a menu to pick one program, try this example:

https://docs.pybricks.com/en/latest/tools/index.html#pybricks.tools.hub_menu

**Reply by AyadTatari** [reply]
Thanks but I need it to be like the code goes off by me clicking on the main button but it only goes off as im clicking on it to save more time if you get what I mean.

**Reply by laurensvalk** [reply]
Yes, the program will start by pressing the hub button. You don't have to press the play button in the app.
